domainlab/utils/hyperparameter_retrieval.py

```diff
@@ -0,0 +1,18 @@
+"""
+retrieval for hyperparameters
+"""
+
+def get_gamma_reg(args, model_name):
```

Comment:
it looks like the correct argument name should be model_or_trainer_name instead of model_name? @MatteoWohlrapp

Reply:
You are right, will change that.

Reply:
I named it component_name to generalize.